.github/workflows/ci.yaml

```diff
@@ -101,6 +101,15 @@ jobs:
           ./**/test-results.xml
           ./**/hspec-results.md
 
+
+    # NOTE: This depends on the path used in hydra-cluster e2e tests
+    - name: 💾 Upload logs
+      if: always()
```

Comment:
why always and not just in case of failure?
is there any value by keeping the logs on succeed?